microcosm_pubsub/codecs.py

```diff
@@ -85,7 +85,11 @@ def decode(self, message):
         Uses the appropriate coded to read JSON.
 
         """
-        dct = loads(message)
+        try:
+            dct = loads(message)
+        except TypeError:
+            # message is already a dict
```

Comment:
In this case, we can use `isinstance` instead.

Reply:
@pelegdvir some folks would say that using `try`/`except` is more Pythonic ("ask for forgiveness" and all that)